features/support/hooks.rb

```diff
@@ -6,6 +6,10 @@
   Capybara.page&.driver&.quit
 end
 
+After do
+  ActiveRecord::Base.clear_active_connections!
```

Comment:
actually this method doesn't exist with Rails 8.1 but I assume it will be easy to fix when time comes.